internal/operator-controller/action/restconfig.go

```diff
@@ -15,6 +15,9 @@ import (
 func ServiceAccountRestConfigMapper(tokenGetter *authentication.TokenGetter) func(ctx context.Context, o client.Object, c *rest.Config) (*rest.Config, error) {
 	return func(ctx context.Context, o client.Object, c *rest.Config) (*rest.Config, error) {
 		cExt := o.(*ocv1.ClusterExtension)
+		if cExt.Spec.ServiceAccount == nil {
+			return rest.CopyConfig(c), nil
+		}
```

Comment:
So, can we create one to bind if the user does not provide an SA? 
Is that?

Reply:
the point of this PR was just to demo the kind of UX I'm after. It just made it so that you could stamp out a ClusterExtension and everything would "just work". I didn't really think about all the odds and ends. I think the idea here was: olm is admin, if user doesn't specify an sa, just use olm's.